src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -3251,15 +3249,6 @@ static Widget _topToolbarWidgets[] = {
         }
     };
 
-    static void ScenarioSelectCallback(const utf8* path)
-    {
-        WindowCloseByClass(WindowClass::EditorObjectSelection);
-        GameNotifyMapChange();
```

Comment:
These callbacks exist for scripting purposes. Are we sure they can just be removed?

Reply:
Quite sure they can’t.

Reply:
The new callback it is calling [is referenced here](https://github.com/OpenRCT2/OpenRCT2/blob/cd528665e64d252dbfe754268432d32df569002a/src/openrct2/Game.cpp#L625-L632). The changes will call this instead of the callback in the TopToolbar. If this will still have an affect on scripting then I can cancel this PR but it looks like this callback is only referenced in the location I changed.

Reply:
@tmatale's analysis seem correct. When in `PromptMode::SaveBeforeNewGame` the save dialog will eventually fall into `GameLoadOrQuitNoSavePrompt` which will call the aforementioned function

Reply:
Very well, I’ll go with that.